static/js/segmentForm.js

```diff
@@ -2,33 +2,13 @@
 /* eslint-disable */
 
 !function(){
-  /**
-   * Add the segment loader to the page
-   */
-  const script = document.createElement('script');
-  script.innerHTML = '! function() { var analytics = window.analytics = window.analytics || []; if (!analytics.initialize)' +
-       'if (analytics.invoked) window.console && console.error && console.error("Segment snippet included twice.");' +
-       'else { analytics.invoked = !0;' +
-         'analytics.methods = ["trackSubmit", "trackClick", "trackLink", "trackForm", "pageview", "identify", "reset", "group", "track", "ready", "alias", "debug", "page", "once", "off", "on"];' +
-         'analytics.factory = function(t) {' +
-         '  return function() {' +
-         '    var e = Array.prototype.slice.call(arguments);' +
-         '    e.unshift(t);' +
-         '    analytics.push(e);' +
-         '    return analytics;' +
-         '  }' +
-         '};' +
-         'for (var t = 0; t < analytics.methods.length; t++) {' +
-         '  var e = analytics.methods[t];' +
-         '  analytics[e] = analytics.factory(e)' +
-         '}' +
-         'analytics.SNIPPET_VERSION = "4.1.0";' +
-      'analytics.page();' +
-    '}}();';
-
-  document.getElementsByTagName('head')[0].appendChild(script);
 
   function testSegment() {
+    // Only test for segment (and open the modal) when there's a form on the page
+    if (!document.forms.length) {
+      return
```

Comment:
super nitty, but there should be a semicolon at the end of this line. Don't waste your time if you don't end up pushing up more changes. I can catch it on a future commit

Reply:
Done